src/main/webapp/i18n/de/metis.json

```
@@ -104,6 +104,11 @@
             "answerPost": {
                 "created": "Antwort erfolgreich erstellt",
                 "deleted": "Antwort erfolgreich gelöscht"
+            },
+            "userAuthorityTooltips": {
+                "instructor": "Lehrkraft: Verantwortlich für die Kursdurchführung.",
+                "ta": "Tutor:in (Lehrassistenz): Verantwortlich für die Unterstützung der Studierenden bei ihren Fragen zum Kurs und zu den Übungen.",
```

```suggestion
                "ta": "Tutor:in (Lehrassistenz): Verantwortlich für die Unterstützung der Studierenden bei ihren Fragen zum Kurs und zu den Übungen.",
```
```suggestion
                "ta": "Tutor:in (Lehrassistent): Verantwortlich für die Unterstützung der Studierenden bei ihren Fragen zum Kurs und zu den Übungen.",
```
but i am not quite sure, both sounds strange in German

---

I'm not an expert in German, so maybe others will have better suggestions 🤞🏻

---

Since you left out the explaining parentheses in the other roles. I'd also leave them out here

---

@krusche also thinks the same. The term is removed.